tests/Unit/IO/Test_VolumeData.py

```diff
@@ -10,160 +10,135 @@
 import numpy.testing as npt
 
 
-class TestIOH5VolumeData(unittest.TestCase):
+class TestVolumeDataWriting(unittest.TestCase):
     # Test Fixtures
     def setUp(self):
-        # The First tests involve inserting vol files, the h5 file for
-        # these will be deleted and recreated for each test as needed.
-        # The other tests use a volume data file written using
+        # The tests in this class involve inserting vol files, the h5 file
+        # will be deleted and recreated for each test
+        self.file_name = os.path.join(Informer.unit_test_path(),
+                                      "IO/python_test.h5")
```

Comment:
ping https://github.com/sxs-collaboration/spectre/pull/2318#discussion_r450336249. Just name the generated files like the test class that generates them, so they are unique. I.e. name this one `TestVolumeDataWriting.h5` and the other one below `TestVolumeData.h5`.